packages/js/client/src/__tests__/e2e/test-cases.ts

```diff
@@ -845,128 +844,58 @@ export const runSimpleStorageTest = async (
   expect(set.data).toBeTruthy();
   expect(set.data?.indexOf("0x")).toBeGreaterThan(-1);
 
-  const getWithStringType = await client.query<{
-    getData: number;
-    secondGetData: number;
-    thirdGetData: number;
-  }>({
+  const getDataResult = await client.invoke<number>({
     uri: wrapperUri,
-    query: `
-        query {
-          getData(
-            address: "${address}"
-            connection: {
-              networkNameOrChainId: "testnet"
-            }
-          )
-          secondGetData: getData(
-            address: "${address}"
-            connection: {
-              networkNameOrChainId: "testnet"
-            }
-          )
-          thirdGetData: getData(
-            address: "${address}"
-            connection: {
-              networkNameOrChainId: "testnet"
-            }
-          )
-        }
-      `,
-  });
-
-  expect(getWithStringType.errors).toBeFalsy();
-  expect(getWithStringType.data).toBeTruthy();
-  expect(getWithStringType.data?.getData).toBe(55);
-  expect(getWithStringType.data?.secondGetData).toBe(55);
-  expect(getWithStringType.data?.thirdGetData).toBe(55);
-
-  const getWithUriType = await client.query<{
-    getData: number;
-    secondGetData: number;
-    thirdGetData: number;
-  }>({
-    uri: wrapperUri,
-    query: `
-        query {
-          getData(
-            address: "${address}"
-            connection: {
-              networkNameOrChainId: "testnet"
-            }
-          )
-          secondGetData: getData(
-            address: "${address}"
-            connection: {
-              networkNameOrChainId: "testnet"
-            }
-          )
-          thirdGetData: getData(
-            address: "${address}"
-            connection: {
-              networkNameOrChainId: "testnet"
-            }
-          )
-        }
-      `,
+    method: "getData",
+    args: {
+      address,
+      connection: {
+        networkNameOrChainId: "testnet",
+      },
+    },
   });
 
-  expect(getWithUriType.errors).toBeFalsy();
-  expect(getWithUriType.data).toBeTruthy();
-  expect(getWithUriType.data?.getData).toBe(55);
-  expect(getWithUriType.data?.secondGetData).toBe(55);
-  expect(getWithUriType.data?.thirdGetData).toBe(55);
+  expect(getDataResult.error).toBeFalsy();
+  expect(getDataResult.data).toEqual(55);
 };
 
 export const runSimpleEnvTest = async (
   client: PolywrapClient,
   wrapperUri: string
 ) => {
-  const queryGetEnv = await client.query({
+  const queryGetEnv = await client.invoke({
```

Comment:
`invokeGetEnv` instead of `queryGetEnv` maybe?

Reply:
I am seeing that this is done in multiple parts, so I am not commenting on all of those but rather making a suggestion here to change it in every part `query` is mentioned :P

Reply:
I prefer `getEnvResult`